seqr/views/utils/anvil_metadata_utils.py

```diff
@@ -96,22 +112,23 @@ def parse_anvil_metadata(projects, max_loaded_date, user, add_row, omit_airtable
         })
         family_data_by_id[family_id] = f
 
-    samples_by_family_id = defaultdict(list)
+    individuals_by_family_id = defaultdict(list)
```

Comment:
can you remind me of the difference btw samples and individuals, and why you changed this data structure to the latter?

Reply:
Individuals represent a person. Samples represent a biological sample that was processed for genetic information. There is a one-to-many relationship between Individuals and Samples, and not all Individuals have any Samples. The change is because this helper was previously used for reports that only included Individuals with loaded data, so we were fetching the sample models and then using the associated Individual when needed. But this report needs to include Individuals that do not have  and loaded data (i.e no associated Samples) so that approach results in data being omitted. We actually were already storing `individual_samples` as a mapping between individuals and their sample (for the purposes of this report we only use the latest Sample for each Individual), so changing this structure to keep track of all Individuals instead of Samples means no one is left out, but we can still easily get the sample in cases where it is needed

Reply:
this clears things up, thank you!